### PR TITLE
Format the Containers submodule

### PR DESCRIPTION
**Pull request by odow**
x-ref: #2292, #2383

### CODE REVIEW

**Comment by mlubin** [comment]
If we doing this systematically, the formatting script should be checked in also.

**Comment by odow** [comment]
Added a TOML configuration file, so now you can just go `using JuliaFormatter; format(dir)` and it will pick up the correct settings.

**Review by mlubin — APPROVED** [review]

File: src/Containers/Containers.jl [hunk 1]
```diff
@@ -26,13 +26,16 @@ export DenseAxisArray, SparseAxisArray
 include("DenseAxisArray.jl")
 include("SparseAxisArray.jl")
 
-Base.eachindex(g::Base.Generator{<:Union{DenseAxisArray, SparseAxisArray}}) = eachindex(g.iter)
+Base.eachindex(g::Base.Generator{<:Union{DenseAxisArray,SparseAxisArray}}) =
+    eachindex(g.iter)
```
Comment (blegat) [inline src/Containers/Containers.jl:30]: This is not allowed by our style guide, we should use `function` if ot is multiline

Comment (odow) [reply src/Containers/Containers.jl:30]: The formatter only provides an option to re-write every single-line function, not an option to rewrite only those that are longer than the line limit.

Merging for now, and we can fix any offending examples by hand.

Comment (odow) [reply src/Containers/Containers.jl:30]: I was wrong: https://github.com/jump-dev/JuMP.jl/pull/2385

**Pull request merged**